components/molecule/dropdownOption/src/_settings.scss

```diff
@@ -0,0 +1,3 @@
+$c-dropdown-option-disabled: #dcdcdc !default;
```

Comment:
this is a lighten gray_l3, please check the color-variation function

I'm not sure if it should be `color-variation($c-gray, 3)`